lab-james/routes/game-routes.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
```

Comment:
So which ever is your second resource has to have: 'It must have a relationship of many to one with the resource from from lab-13'. Per lab instructions. So an endpoint would be like /gameConsole/:gameConsoleId/game/:id'.